pkgs/by-name/sl/slurm-nm/package.nix

```
@@ -30,7 +30,10 @@ stdenv.mkDerivation rec {
     description = "Generic network load monitor";
     homepage = "https://github.com/mattthias/slurm";
     license = licenses.gpl2Plus;
-    platforms = [ "x86_64-linux" ];
+    platforms = [
+      "x86_64-linux"
+      "aarch64-linux"
+    ];
```

```suggestion
    platforms = platforms.unix;
```

https://github.com/mattthias/slurm#slurm-was-tested-on

Don't only mark platforms you tested. Assume all platforms mentioned by upstream works. Only mark badPlatforms when you know it doesn't work.

---

@Aleksanaa I'm not the initall author of the package so i dont know why it only had contained "x86_64-linux" so thats why i did only add the arch i needed ... and the one i could test :D 

but I sure can apply yours if thats what blocks the merge ;)